src/ui/DynamicFacet/DynamicFacetValues/DynamicFacetValue.ts

```diff
@@ -9,18 +9,21 @@ export interface IDynamicFacetValue {
   value: string;
   state: FacetValueState;
   numberOfResults: number;
+  position: number;
 }
 
 export class DynamicFacetValue implements IDynamicFacetValue {
   public value: string;
   public state: FacetValueState;
   public numberOfResults: number;
+  public position: number;
   private renderer: DynamicFacetValueRenderer;
 
-  constructor({ value, state, numberOfResults }: IDynamicFacetValue, private facet: DynamicFacet) {
+  constructor({ value, state, numberOfResults, position }: IDynamicFacetValue, private facet: DynamicFacet) {
```

Comment:
Instead of deconstructing each property and setting it to a class property in the constructor, if you add the `public` keyword next to the constructor arg name, which will make the options object available. e.g.

```
constructor(public dynamicFacetValue: IDynamicFacetValue, private facet: DynamicFacet) {}
```
You would not need to add a property, deconstruct, and then set the property. Seems more concise. However, if there are properties that you want to keep `private`, then the current approach is better/gives you more control.